cypress/integration/reportingView.spec.js

```diff
@@ -57,11 +57,6 @@ describe('Reporting Download', () => {
       cy.get('.MuiAlert-message').contains('Report generated successfully!').should('be.visible');
     });
 
-    it('Should be able to download Program Monitoring report', () => {
-      clickReportButton('Download program monitoring report');
-      cy.get('.MuiAlert-message').contains('Report generated successfully!').should('be.visible');
-    });
```

Comment:
What is the purpose of removing this test?

Reply:
that button was removed in [PR-1026](https://github.com/bcgov/hcap/pull/1026/files) and the test was never removed